build.gradle

```diff
@@ -9,6 +9,21 @@ reckon {
 
 allprojects {
     pluginManager.withPlugin('java') {
```

Comment:
Shouldn't this also be the `java-library` plugin then?

Reply:
In practice there is no difference. But it is updated now.